lib/src/layer/marker_layer.dart

```diff
@@ -107,18 +107,16 @@ class MarkerLayer extends StatelessWidget {
   final MapState map;
   final Stream<Null> stream;
 
-  MarkerLayer(this.markerOpts, this.map, this.stream)
-      : super(key: markerOpts.key);
-
-  bool _boundsContainsMarker(Marker marker) {
-    var pixelPoint = map.project(marker.point);
-
-    final width = marker.width - marker.anchor.left;
-    final height = marker.height - marker.anchor.top;
+  // Note: I don't know if it's okay to store mutable stuff like this in stless
```

Comment:
It's not; this will be discarded anytime this widget is rebuilt.

Reply:
Convert this into a StatefulWidget and clear the pxCache and lastZoom when the `didUpdateWidget` lifecycle method is called.

Reply:
@TheLastGimbus Did you the the request for change from @johnpryan ?